commands/sense/README.md

```diff
@@ -45,8 +48,6 @@ export default {
 nebula sense --ext def.js
 ```
 
-The command generates all required files into the specified `--output` folder. You can the zip the folder and upload it as an extension on [Qlik Sense Enterprise for Windows](https://help.qlik.com/en-US/sense-developer/April2020/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) or [Qlik Sense Enterprise on Kubernetes](https://help.qlik.com/en-US/sense-admin/April2020/Subsystems/DeployAdministerQSE/Content/Sense_DeployAdminister/QSEoK/Administer_QSEoK/mc-extensions.htm)
-
 ### Meta
 
 You can add more meta about the extension by providing a `.json` formatted file with `--meta`:
```

Comment:
Should add an example json file here, and explain that it ends up in the qext file used by sense.

Reply:
I did not change this section. An example JSON file was presented before. 
Great. Gonna add that explanation about meta.